src/components/char-check-limit/_macro.spec.js

```diff
@@ -58,4 +58,28 @@ describe('FOR: Macro: CharCheckLimit', () => {
             });
         });
     });
+    describe('WHEN: variant is set to word', () => {
+        const $ = cheerio.load(renderComponent('char-check-limit', EXAMPLE_WORD_LIMIT));
+
+        test('THEN: passes jest-axe checks', async () => {
+            const results = await axe($.html());
+
+            expect(results).toHaveNoViolations();
+        });
+        test('THEN: has the provided id attribute', () => {
+            expect($('.ons-input__limit').attr('id')).toBe('example-word-limit');
+        });
```

Comment:
This test seems to be in the wrong place, the id attribute isn't set because the variant is set to word which is what this seems to suggest